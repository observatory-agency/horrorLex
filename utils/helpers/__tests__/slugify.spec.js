const slugify = require('../slugify');

describe('slugify', () => {
  it('should match the snapshot', () => {
    const title = 'Encyclopedia of the Vampire: The Living Dead in Myth, Legend, and Popular Culture';
    expect(slugify(title)).toMatchSnapshot();
  });
});
