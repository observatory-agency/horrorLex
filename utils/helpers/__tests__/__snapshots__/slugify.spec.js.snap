// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`slugify should match the snapshot 1`] = `"encyclopedia-of-the-vampire-the-living-dead-in-myth-legend-and-popular-culture"`;
